### DISCUSSION TITLE
Reset AP only

### DISCUSSION THREAD

**Discussion by dac2020** [arendst/Tasmota#16679] [Q&A] [answered]
Is there a way to keep all of the settings but only reset the access point?
I want to be able to send some devices to some friends but I don't want them to have to configure everything.
I'd like to send them the device, they see it on their phone or laptop in AP mode. They set the AP and once it has wifi, it starts communicating with my broker.

I see the AP commands in the commands list but there is not much info in terms of what I'd like to do.

**Comment by sfromis** [comment, marked as answer]
You can clear the two sets of SSID/Password using those commands and set `WifiConfig 2` for it to start in AP mode.

**Reply by dac2020** [reply]
Hi,

And that will only lose the AP info and nothing else?
I love well thought out software. Very nice.
I'll give that a try.

**Reply by dac2020** [reply]
Works perfectly.

2 = set [Wi-Fi Manager](https://github.com/tzapu/WiFiManager/blob/master/README.md#how-it-works) as the current configuration tool and start Wi-Fi Manager (web server at 192.168.4.1) for 3 minutes, then reboot and try to connect Wi-Fi network

Any way of extending the 3 minutes in case someone misses the timeline and doesn't get it connected in time?

**Reply by sfromis** [reply]
That was why I suggested you to clear the current set(s) of SSID/Password. Then it has no network to try to connect to.

**Reply by dac2020** [reply]
I must have missed that and yes, that would be perfect since it could remain in AP mode until configured which is what I'd like.

Thank you.
